imod_coupler/drivers/dfm_metamod/dfm_metamod.py

```diff
@@ -38,30 +40,14 @@ class DfmMetaMod(Driver):
     max_iter: NDArray[Any]  # max. nr outer iterations in MODFLOW kernel
     delt: float  # time step from MODFLOW 6 (leading)
 
-    mf6_head: NDArray[Any]  # the hydraulic head array in the coupled model
-    mf6_recharge: NDArray[Any]  # the coupled recharge array from the RCH package
-    mf6_storage: NDArray[Any]  # the specific storage array (ss)
-    mf6_has_sc1: bool  # when true, specific storage in mf6 is given as a storage coefficient (sc1)
-    mf6_area: NDArray[Any]  # cell area (size:nodes)
-    mf6_top: NDArray[Any]  # top of cell (size:nodes)
-    mf6_bot: NDArray[Any]  # bottom of cell (size:nodes)
-
-    mf6_sprinkling_wells: NDArray[Any]  # the well data for coupled extractions
-    msw_head: NDArray[Any]  # internal MetaSWAP groundwater head
-    msw_volume: NDArray[Any]  # unsaturated zone flux (as a volume!)
-    msw_storage: NDArray[Any]  # MetaSWAP storage coefficients (MODFLOW's sc1)
-    msw_time: float  # MetaSWAP current time
-
-    dflowfm_stage: NDArray[Any]
-
-    # dictionary with mapping tables for mod=>msw coupling
-    map_mod2msw: Dict[str, csr_matrix] = {}
-    # dictionary with mapping tables for msw=>mod coupling
-    map_msw2mod: Dict[str, csr_matrix] = {}
-    # dict. with mask arrays for mod=>msw coupling
-    mask_mod2msw: Dict[str, NDArray[Any]] = {}
-    # dict. with mask arrays for msw=>mod coupling
-    mask_msw2mod: Dict[str, NDArray[Any]] = {}
+    mf6_head: NDArray[Any]  # the hydraulic head array in the groundwater model
+    mf6_river_stage: NDArray[Any]  # the river stage array in the groundwater model
+
+    dflowfm_1d_stage: NDArray[
+        Any
```

Comment:
This `Any` is still a relic from before I know how to properly use typing with numpy.
You should, depending on whether it is a float or integer, use `np.float_` or `np.int_` instead

Reply:
done

Reply:
done